src/plugins/checkboxes/checkbox.scss

```diff
@@ -0,0 +1,137 @@
+.provisional {
```

Comment:
This could potentially affect other provisional features related to forms, like wb-postback. Since wb-postback is provisional the form will get those new styles and it could be unwanted.

Reply:
I will scope it a little bit more:
```suggestion
.provisional {
   .chckbxrdio-grp label {
```
Would that make sense?

Reply:
Still too broad